boot/BootFromDevice.c

```diff
@@ -80,7 +80,7 @@ CONFIGENTRY* DetectSystemNative(int drive, int partition) {
 	char* szGrub;
 
 	szGrub = InitGrubRequest(GRUB_REQUEST_SIZE, drive, partition);
-	config = LoadConfigNative(szGrub);
+	config = DetectLinuxNative(szGrub);
```

Comment:
What is native? Flash? HDD?

Reply:
Native HDD partition (i.e. not FATX partition, but one of PC compatible ones).